addon/locale/pt_BR/LC_MESSAGES/nvda.po

```diff
@@ -0,0 +1,92 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the snippetsForNVDA package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: snippetsForNVDA 1.0.0\n"
+"Report-Msgid-Bugs-To: nvda-translations@groups.io\n"
+"POT-Creation-Date: 2020-06-27 14:02-0300\n"
+"PO-Revision-Date: 2020-06-27 16:38-0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: pt_BR\n"
+
+#. Translators: This message is displayed when text is saved on a memory slot.
+#: addon\globalPlugins\snippetsForNVDA\__init__.py:35
+msgid "Saved"
+msgstr "Salvo"
+
+#. Translators: the documentation of the save to memory slot command, displayed on the input help mode.
+#: addon\globalPlugins\snippetsForNVDA\__init__.py:40
+msgid "When pressed, this key saves the selected text to this memory slot."
+msgstr ""
+"Quando pressionada, esta tecla salva o texto selecionado nessa posição de "
+"memória."
+
+#. Translators: The message displayed when the user pasted this memory slot to an edit field.
+#: addon\globalPlugins\snippetsForNVDA\__init__.py:52
+#, python-brace-format
+msgid "Pasted {data}"
+msgstr "Colado {data}"
+
+#. Translators: The message when the user checks a memory slot but there is no data in it
+#: addon\globalPlugins\snippetsForNVDA\__init__.py:61
+msgid "No data at this position"
+msgstr "Não há dados nessa posição"
+
+#. Translators: the documentation of the speak and copy memory slot command, displayed on the input help mode.
+#: addon\globalPlugins\snippetsForNVDA\__init__.py:65
+msgid ""
+"Pressing this key combination once , the content of this memory slot will be "
+"spoken.\n"
+"Pressing it twice in a row, the content of this memory slot will be pasted "
+"to the running application."
+msgstr ""
+"Pressionando essa combinação de teclas uma vez, o conteúdo dessa posição de "
+"memória será falado.\n"
+"Pressionando-o duas vezes seguidas, o conteúdo dessa posição de memória será "
```

Comment:
Talvez deixar claro que precisam ser duas vezes rapidamente? Se for o caso, alterar também em inglês.

Reply:
Sim, isso faz sentido, vou alterar em inglês também.